Develop Library mock up

pva701 opened the pull request:
## Description

Representations for all four entities were implemented.

## Related issue(s)

<!--
- Short description of how the PR relates to the issue, including an issue link.
For example
- Fixed #100500 by adding lenses to exported items

Write 'None' if there are no related issues (which is discouraged).
-->

https://issues.serokell.io/issue/EDNA-31

## :white_check_mark: Checklist for your Pull Request

<!--
Ideally a PR has all of the checkmarks set.

If something in this list is irrelevant to your PR, you should still set this
checkmark indicating that you are sure it is dealt with (be that by irrelevance).

If you don't set a checkmark (e. g. don't add a test for new functionality),
you must be able to justify that.
-->

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [x] My code complies with the [style guide](../tree/master/docs/code-style.md).

pva701 committed:
Develop projects layout according to the design mock up

Problem: we need to have better correspondence
with mock up design.

Solution: design project table.

pva701 committed:
Add Compounds and Targets tables

Problem: we don't have Compounds and Targets tab layouts.

Solution: use table for Compounds and Targets tab.

pva701 committed:
Change methodologies view according to the mock up

Problem: we don't have methodologies layout.

Solution: implement design of this tab.

sashasashasasha151 commented on frontend/src/pages/library/LibraryPage.tsx:
```diff
@@ -79,34 +76,111 @@ function EntitiesTab({ render, renderAddButton }: EntitiesTabProps) {
+  const methodologies = useRecoilValue(methodologiesAtom);
+  return (
+    <>
+      {methodologies.map(m => (
+        <MethodologyPlate title={m.name} description={m?.description} />
+      ))}
```
![image](https://user-images.githubusercontent.com/25207759/111998686-51f91680-8b2d-11eb-95f3-523098831a0e.png)

pva701 replied:
huita

gromakovsky reviewed (approved):
Seems to work fine.

pva701 committed:
Make table 100% wide in firefox

Problem: library tables is not 100% wide in firefox.

Solution: add property to make so.
